packages/ketcher-core/src/domain/entities/BaseMonomer.ts

```diff
@@ -70,6 +70,22 @@ export class BaseMonomer extends DrawingEntity {
     return undefined;
   }
 
+  public get R1AttachmentPoint() {
```

Comment:
Could you please set return type here AttachmentPointName? It exists packages/ketcher-core/src/domain/types/monomers.ts